pip_update.sh

```diff
@@ -1,9 +1,25 @@
 #!/bin/bash
+
+POSITIONAL_ARGS=()
```

Comment:
Add a comment with instructions on how to use this

Reply:
added